plugin-trino/src/main/java/org/apache/ranger/authorization/trino/authorizer/RangerSystemAccessControl.java

```diff
@@ -42,7 +42,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.common.collect.ImmutableList;
+import org.apache.hadoop.thirdparty.com.google.common.collect.ImmutableList;
```

Comment:
In slack you wrote:

> I've kept the old guava in Presto and Trino plugins because it's
>  using a different guava version. I don't think that this is going to
>  affect us. We don't setup Ranger with Trino.

I'm concerned about this.  I see that some of the trino files have changed:
```
plugin-trino/src/main/java/org/apache/ranger/authorization/trino/authorizer/RangerSystemAccessControl.java:45:import org.apache.hadoop.thirdparty.com.google.common.collect.ImmutableList;
plugin-trino/src/test/java/org/apache/ranger/authorization/trino/authorizer/RangerSystemAccessControlTest.java:20:import org.apache.hadoop.thirdparty.com.google.common.collect.ImmutableSet;
ranger-trino-plugin-shim/src/main/java/org/apache/ranger/authorization/trino/authorizer/RangerSystemAccessControlFactory.java:24:import static org.apache.hadoop.thirdparty.com.google.common.base.Throwables.throwIfUnchecked;
```
And it isn't clear to me the trino pom files are correct:

This one:
```
ranger/plugin-trino/pom.xml
```
seems to use the parent pom's guava version
While this one:
```
ranger-trino-plugin-shim/pom.xml
```
uses it's own.


I think the cleanest thing to do is to get rid of these entirely:
```
<google.guava.version>27.0-jre</google.guava.version>
<presto.guava.version>26.0-jre</presto.guava.version>
<trino.guava.version>26.0-jre</trino.guava.version>
```

and just have:
```
<hadoop-thirdparty-guava.version>1.1.1</hadoop-thirdparty-guava.version>
```

Otherwise it will be confusing.